docs/manage/reference/viam-agent/manage-viam-agent.md

```diff
@@ -114,6 +114,12 @@ On Windows, you can manage `viam-agent` using the Services GUI or the command li
 {{% /tab %}}
 {{< /tabs >}}
 
+To uninstall `viam-agent`, run the following command:
+
+```sh {class="command-line" data-prompt="$"}
+sc delete viam-agent
```

Comment:
notes on this:
1. has to happen in an administrator prompt (start -> cmd -> right click -> run as administrator), otherwise you should get access denied. this may not be necessary on a system with developer mode
2. if agent is running, the service won't be deleted until the user stops it manually; could tell people to run `sc stop viam-agent` first
3. this won't delete any files, it will just stop agent from running

Reply:
Are there files aside from the installer and the viam.json file? I suppose everything that got installed... Is there anything we can do aside from tell people to remove whichever folder they are in?

Reply:
(oops missed last week)

the user can delete the whole `/opt/viam` folder and the viam dotdir folder (`C:\Windows\system32\config\systemprofile\.viam`)

we don't use a normal MSI because symlinking is thorny with MSI, but if we did, it would have an entry in the 'add/remove programs' control panel page, and would clean up any folders it installed. (but we don't have this)